src/query/queryWithCursor.spec.ts

```diff
@@ -60,17 +60,6 @@ describe('queryWithCursor', () => {
     ).rejects.toThrowError('Partition key condition can only be a string');
   });
 
-  it('should throw an error when max limit is exceeded', async () => {
```

Comment:
Shouldn't we test for when the limit is **100000** ?

Reply:
if limit is 10000, then we should fetch 10000 records